habitat-baselines/habitat_baselines/config/rearrange/rl_hierarchical_oracle_nav_human.yaml

```diff
@@ -0,0 +1,58 @@
+# @package _global_
```

Comment:
I don't like creating another one of these configs. Is there any way you can unify this with the existing `rl_hierarchical` config and have argument overrides or compose with another config to configure the human? At the very least, you should derive from the existing `rl_hierarchical_oracle_nav` config.